src/items.cpp

```diff
@@ -105,12 +105,12 @@ Item* newItem(ItemType type, Status status, Sint16 beatitude, Sint16 count, Uint
 		}
 		else if ( multiplayer != CLIENT )
 		{
-			for ( int i = 0; i < MAXPLAYERS; ++i )
```

Comment:
Would rather auto only be used for std containers such as vectors/lists instead of array[]

Reply:
Fixed in f76c8571